keda/values.yaml

```diff
@@ -57,11 +57,22 @@ serviceAccount:
   # Annotations to add to the service account
   annotations: {}
 
-# Set to the value of the Azure Active Directory Pod Identity
-# This will be set as a label on the KEDA Pod(s)
 podIdentity:
   activeDirectory:
+    # Set to the value of the Azure Active Directory Pod Identity
+    # This will be set as a label on the KEDA Pod(s)
     identity: ""
+  azureWorkload:
+    # Set to true to enable Azure Workload Identity usage.
```

Comment:
```suggestion
    # Set to the value of the Azure Active Directory Pod Identity
    # See https://keda.sh/docs/concepts/authentication/#azure-pod-identity
    # This will be set as a label on the KEDA Pod(s)
    identity: ""
  azureWorkload:
    # Set to true to enable Azure Workload Identity usage.
    # See https://keda.sh/docs/concepts/authentication/#azure-workload-identity
```